DigitalLearningSolutions.Data/Services/PasswordResetService.cs

```diff
@@ -89,22 +89,26 @@ END CATCH
         private Email GeneratePasswordResetEmail(string emailAddress, string resetHash, string firstName, string baseUrl)
         {
             UriBuilder resetPasswordUrl = new UriBuilder(baseUrl);
+            if (!resetPasswordUrl.Path.EndsWith('/'))
```

Comment:
Do we know why the path doesn't end with "/" sometimes, but does in other times (the emails work ok in staging)? Is the AppRootPath consistent across the environments?

Reply:
It's because the UAT has some path already, i.e. at www.example.com/UAT rather than at test.example.com. When we create the UriBuilder with baseUrl with a url like this we get:

- Host: www.example.com
- Path: /UAT

rather than
- Host: test.example.com
- Path: /

Reply:
Ah, that makes sense! 👍 😅